python/ml4ir/applications/ranking/tests/data/configs/feature_config_integration_test.yaml

```diff
@@ -0,0 +1,170 @@
+query_key: 
+  name: query_id
+  node_name: query_id
+  trainable: false
+  dtype: string
+  log_at_inference: true
+  feature_layer_info:
+    type: numeric
+    shape: null
+  serving_info:
+    name: queryId
+    required: false
+    default_value: ""
+  tfrecord_type: context
+rank:
+  name: rank
+  node_name: rank
+  trainable: false
+  dtype: int64
+  log_at_inference: true
+  feature_layer_info:
+    type: numeric
+    shape: null
+  serving_info:
+    name: originalRank
+    required: true
+    default_value: 0
+  tfrecord_type: sequence
+label:
+  name: clicked
+  node_name: clicked
+  trainable: false
+  dtype: int64
+  log_at_inference: true
+  feature_layer_info:
+    type: numeric
+    shape: null
+  serving_info:
+    name: clicked
+    required: false
+    default_value: 0
+  tfrecord_type: sequence
+features:
+  - name: text_match_score
+    node_name: text_match_score
+    trainable: true
+    dtype: float
+    log_at_inference: true
+    feature_layer_info:
+      type: numeric
+      shape: null
+    serving_info:
+      name: textMatchScore
+      required: true
+      default_value: 0.0
+    tfrecord_type: sequence
+  - name: page_views_score
+    node_name: page_views_score
+    trainable: true
+    dtype: float
+    log_at_inference: true
+    feature_layer_info:
+      type: numeric
+      shape: null
+      fn: tf_native_op
+      args:
+        ops:
+          - fn: tf.math.add
+            kwargs:
+              y: 1.
+          - fn: tf.clip_by_value
+            kwargs:
+              clip_value_min: 0.
+              clip_value_max: 10.
```

Comment:
@jakemannix FYI I've made a disruptive change to the feature layer here, which should be captured in the integration test.